### PR TITLE
Closes #53: Write a simple spark example in core-examples 

### LINKED ISSUE #53: Write a simple spark example in core-examples

### LINKED ISSUE #60: Upgrade versions of hadoop libs to v2 (from mr1)

### PR DESCRIPTION
**Pull request by lukdumi**
Actually it closes 2 tasks:
Closes #53: Write a simple spark example in core-examples 
Closes #60: Upgrade versions of hadoop libs to v2 (from mr1)

### CODE REVIEW

File: iis-workflows/iis-workflows-core-examples/pom.xml [hunk 1]
```diff
@@ -117,11 +125,7 @@
         <!-- Needed by Avro } -->
         <!-- Needed by our Hadoop Streaming workflow node {-->
         
-        <dependency>
-            <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-streaming</artifactId>
-        </dependency>
-        <!-- Needed by our Hadoop Streaming workflow node }-->
```
Comment (madryk) [inline iis-workflows/iis-workflows-core-examples/pom.xml]: remove also comment above `<!-- Needed by our Hadoop Streaming workflow node {-->`

File: iis-workflows/pom.xml [hunk 2]
```diff
@@ -64,6 +64,9 @@
         <oozie.package.skip.test.jar>true</oozie.package.skip.test.jar>
         
         <output.dir.name>${maven.build.timestamp}</output.dir.name>        
+        
+        <projectTestJarFile>${project.artifactId}-${project.version}-tests.jar</projectTestJarFile>
+        <oozie.use.system.libpath>true</oozie.use.system.libpath>
```
Comment (madryk) [inline iis-workflows/pom.xml]: I can't see where this variable is used

Comment (lukdumi) [inline iis-workflows/pom.xml]: It's used in iis-properties-maven-plugin (in the same pom.xml).

File: iis-workflows/iis-workflows-core-examples/pom.xml (hunk 1 above)
Comment (lukdumi) [inline iis-workflows/iis-workflows-core-examples/pom.xml]: ok

**Pull request closed without merging**